sdk/java-sdk-spring/src/it/java/com/example/wiring/actions/headers/EchoAction.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.wiring.actions.headers;
+
+import com.example.wiring.actions.echo.Message;
+import kalix.javasdk.action.Action;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+/**
+ * Action with the same name in a different package.
+ */
+public class EchoAction extends Action {
+
+  @GetMapping("/echo2/message/{msg}")
+  public Effect<Message> stringMessage(@PathVariable String msg) {
+    return effects().reply(new Message(msg));
+  }
+}
```

Comment:
Isn't this file already in [main](https://github.com/lightbend/kalix-jvm-sdk/blob/dc06f25309d75c55ea1faa76c497def6fc085320/sdk/java-sdk-spring/src/it/java/com/example/wiring/actions/headers/EchoAction.java#L27) ? Or what is the difference here?

Reply:
I think I need to rebase this branch.

Reply:
I did a rebase. Both the EchoAction nor the ProtoDescriptorGenerator weren't something that I added myself. They are not removed for the diff.